code/modules/antagonists/ashwalker/ashwalker.dm

```diff
@@ -28,6 +28,7 @@
 	. = ..()
 	RegisterSignal(owner.current, COMSIG_MOB_EXAMINATE, PROC_REF(on_examinate))
 	owner.teach_crafting_recipe(/datum/crafting_recipe/skeleton_key)
+	owner.add_traits(list(TRAIT_SETTLER), ROLE_ASHWALKER)
```

Comment:
Why is this `add_traits` if there's only 1 trait

Reply:
oop